src/families/osmosis/api/mocks.ts

```diff
@@ -0,0 +1,105 @@
+import { BigNumber } from "bignumber.js";
```

Comment:
Thanks for providing this file, however I believe it should be named `mock.ts` and live in the `bridge` folder instead of the `api` one.